cmd/create/cluster/cmd.go

```diff
@@ -176,6 +180,18 @@ func init() {
 		false,
 		"Use AWS Security Token Service (STS) instead of IAM credentials to deploy your cluster.",
 	)
+	flags.BoolVar(
+		&args.nonSts,
+		"non-sts",
+		false,
+		"Use legacy way of creating clusters (IAM mode).",
+	)
+	flags.BoolVar(
```

Comment:
You can use the same arg to bind it to make an 'alias' so you don't end up with two args